web/gui/dashboard_info.js

```diff
@@ -1311,8 +1311,7 @@ netdataDashboard.context = {
 
     'system.clock_sync_state': {
         info:'<p>The system clock synchronization state as provided by the <a href="https://man7.org/linux/man-pages/man2/adjtimex.2.html" target="_blank">ntp_adjtime()</a> system call. '+
-        'This reflects the status of the system kernel regarding clock synchronization. '+ 
-        'it leads to unpredictable problems. '+
+        'An unsynchronized clock may be the result of synchronization issues by the NTP daemon. '+
```

Comment:
Since `TIME_ERROR` (our unsync status) is `STA_UNSYNC|STA_CLOCKERR`, we could add that it could be "clock hardware fault".

Reply:
Does the new timex collector (migrated to ntp_adjtime( ) ) still return STA_UNSYNC|STA_CLOCKERR as the alarm? I thought it would have been useful to know which of the two is at fault when the error is raised

Reply:
`ntp_adjtime()` is basically the equivalent of `adjtimex()` but differently named and more portable.

```cmd
   ntp_adjtime ()
       The ntp_adjtime() library function (described in the NTP "Kernel
       Application Program API", KAPI) is a more portable interface for
       performing the same task as adjtimex().  Other than the following
       points, it is identical to adjtimex():

       *  The constants used in modes are prefixed with "MOD_" rather
          than "ADJ_", and have the same suffixes (thus, MOD_OFFSET,
          MOD_FREQUENCY, and so on), other than the exceptions noted in
          the following points.

       *  MOD_CLKA is the synonym for ADJ_OFFSET_SINGLESHOT.

       *  MOD_CLKB is the synonym for ADJ_TICK.

       *  The is no synonym for ADJ_OFFSET_SS_READ, which is not
          described in the KAPI.
```